sound/soc/sof/core.c

```diff
@@ -310,6 +310,7 @@ int snd_sof_device_probe(struct device *dev, struct snd_sof_pdata *plat_data)
 	/* check all mandatory ops */
 	if (!sof_ops(sdev) || !sof_ops(sdev)->probe || !sof_ops(sdev)->run ||
 	    !sof_ops(sdev)->block_read || !sof_ops(sdev)->block_write ||
+	    !sof_ops(sdev)->mailbox_read || !sof_ops(sdev)->mailbox_write ||
```

Comment:
This was suggested by @ujfalusi in https://github.com/thesofproject/linux/pull/2914.

![image](https://user-images.githubusercontent.com/62316/129217725-e5dca308-070b-4b4c-8e88-7c47d2c0c60d.png)

This commit is linked with the PR title 'Make Intel IPC stream ops generic' because when moving  `sound/soc/sof/intel/intel-ipc.c ` → ` sound/soc/sof/stream-ipc.c ` we need to use 
`snd_sof_mailbox_read` instead of `snd_sof_dsp_mailbox_read`

Reply:
Agree. We will modify the commit message.

Reply:
We updated the commit message, hopefully it is clearer now.